sdk/core/Azure.Core/src/Pipeline/HttpPipelineBuilder.cs

```diff
@@ -57,7 +57,7 @@ public static HttpPipeline Build(PipelineOptions options, params HttpPipelinePol
         /// <param name="perRetryPolicies">Client provided per-retry policies.</param>
         /// <param name="responseClassifier">The client provided response classifier.</param>
         /// <returns>A new instance of <see cref="HttpPipeline"/></returns>
-        private static HttpPipeline Build(
+        public static HttpPipeline Build(
```

Comment:
We're adding a lot of overloads here, it would be nicer to clean this up when we loop back on what the GA API surface looks like.